src/unregister.test.js

```diff
@@ -583,10 +563,10 @@ describe('unregisterNode()', () => {
     nav.unregisterNode('item1')
 
     expect(nav.currentFocusNodeId).toEqual('item2')
-    expect(onBlur).not.toBeCalled();
+    expect(onBlur).not.toBeCalled()
   })
 
-  test('unregistering a pibling of the focused node should not remove focus - forceRefocus false', () => {
+  test('unregistering a sibling of the focused node should not remove focus - forceRefocus false', () => {
```

Comment:
```suggestion
  test('unregistering a pibling of the focused node should not remove focus - forceRefocus false', () => {
```

This should actually be "pibling"... I was confused as well! https://github.com/bbc/lrud/pull/70#pullrequestreview-494782387